utils/hare-shutdown

```diff
@@ -40,7 +41,12 @@ shutdown_sequence = ('s3service', 'ios', 'confd', 'hax', 'consul')
 
 
 def _setup_logging():
-    logging.basicConfig(level=logging.INFO, format='%(message)s')
+    n = len(sys.argv)
+    if n > 1:
+        if sys.argv[1] == '-v':
```

Comment:
What if `-v` stands not at the first position? What if we add more options? How the end user will know that there is a `-v` option available?

I would prefer using [argparse](https://docs.python.org/3/library/argparse.html) or [click](https://click.palletsprojects.com/en/7.x/) (my favorite one).